src/components/common/progress/ProgressItem.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import ProgressItem from './ProgressItem';
+
+const meta: Meta<typeof ProgressItem> = {
+  title: 'Components/Progress/ProgressItem',
+  component: ProgressItem,
+  argTypes: {
+    index: {
+      control: 'number',
+    },
+    title: {
+      control: 'text',
+    },
+    subTitle: {
+      control: 'text',
+    },
+    content: {
+      control: 'text',
+    },
+    isActive: {
+      control: 'boolean',
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof ProgressItem>;
+
+export const Default: Story = {
+  args: {
+    index: 1,
+    title: '프로그래스 타이틀',
+    subTitle: '서브타이틀',
+    content: '프로그레스 내용',
+    isActive: false,
+  },
+};
+
+export const Primary: Story = {
```

Comment:
```suggestion
export const ProgressItemStory: Story = {
```
또는 모든 결과를 보여줄 경우

```suggestion
export const All: Story = {
```
로도 표현이 될 거 같아요! 요 컨벤션도 회의 후에 같이 맞추면 좋을 거 같아요!

Reply:
좋습니다! 이렇게 표현하는게 훨씬 좋아보이네요!